example/wildcat.config.js

```diff
@@ -115,6 +115,11 @@ const wildcatConfig = {
             // App server port
             port: getPort(process.env.PORT, 3000),
 
+            middleware: [
+                // EXAMPLE:
+                require('./api/sampleServerOnlyApi')
+            ],
+
```

Comment:
Can you add a default to [`packages/react-wildcat/src/utils/templates/wildcat.config.js`](https://github.com/nfl/react-wildcat/blob/master/packages%2Freact-wildcat%2Fsrc%2Futils%2Ftemplates%2Fwildcat.config.js)?